parsers/robot.c

```diff
@@ -48,24 +52,47 @@ static void whitespaceSwap (vString *const s)
                 s->buffer[i] = replaceWith;
 }
 
-static void tagKeywords (const char *const line, const regexMatch *const matches,
+static void changeSection (const char *const line, const regexMatch *const matches,
```

Comment:
You can use strncasecmp declared in main/general.h.
With strncasecmp, you can avoid allocating vstring and copy the bytes.